src/trix/models/composition.coffee

```diff
@@ -478,6 +493,15 @@ class Trix.Composition extends Trix.BasicObject
   notifyDelegateOfInsertionAtRange: (range) ->
     @delegate?.compositionDidPerformInsertionAtRange?(range)
 
+  getPositionInBlock: ->
```

Comment:
_Position_ has a very specific meaning in Trix: an index of a single character in a document. This method returns either a string (`"start"` or `"end"`) or a _range_ (an array of [start, end] positions). Neither of those is a position.

Let’s come up with a better name for this method. How would you describe its purpose, in prose? We can start there and whittle it down.